meta-balena-common/recipes-core/images/balena-image-boot-initramfs.bb

```diff
@@ -4,9 +4,17 @@ BalenaOS initramfs"
 
 PACKAGE_INSTALL = " \
     kexecboot \
-    kexec-tools-klibc \
-    kexecboot-cfg \
+    kexec-tools \
+    base-passwd \
+    busybox \
+    glibc-gconv \
+    glibc-gconv-ibm437 \
+    glibc-gconv-ibm850 \
+    initramfs-module-debug \
+    initramfs-framework-base \
+    udev \
```

Comment:
I didn't expect this change to be merged, it was just meant for development. Maybe make it conditional on `OS_DEVELOPMENT`? Although I'd rather not introduce build variants to be honest - these can be included as part of debugging/development.

Reply:
Can we maybe include this at least for the early versions and then remove it later when minimizing? The footprint is not that big and being able to debug issues by default will be useful.